handbook/engineering/onboarding/engineering-manager-onboarding.md

```diff
@@ -0,0 +1,25 @@
+# Engineering manager onboarding
+
+We expect our engineering managers to drive outcomes, so here are the outcomes we expect of you as you are onboarding at Sourcegraph. You have ownership and autonomy on how to achieve these outcomes. If you aren't confident that you are on the right path, just ask for help from your team and/or manager!
+
+## Week 1
+
+- You have completed [general setup](index.md#getting-set-up).
+- You have scheduled recurring 1-1s with your team and ensured that you are invited to (and made the calendar owner of) all of your team's recurring meetings.
+- You have made 1 commit or PR to our codebase (for example: adding yourself to our handbook, onboarding or product docs improvement).
+
+## Month 1
+
+- You are able to describe the team's high level goals, answer questions about them, and defend them if they are challenged.
+- You are able to describe the individual work that each person on your team is doing, answer questions about that work, and defend why that work is important for the team's stated goals.
+- You are able to describe the career aspirations of each teammate, where they are struggling, and where they are thriving.
+- Your manager, your peer managers, and your teammates report that they are happy and relieved to have you on the team because you have already had a positive impact.
```

Comment:
```suggestion
- Your manager, your peer managers, and your teammates report that they are happy and relieved to have you on the team because you have already had a positive impact.
- You understand who are your stakeholders and have meet with them to understand their priorities.
```